sdk/rust/oak/src/lib.rs

```diff
@@ -505,7 +505,7 @@ pub fn run_event_loop<T: crate::io::Decodable, N: Node<T>>(
                         );
                         return;
                     }
-                    ErrBadHandle | ErrChannelClosed | ErrPermissionDenied => {
+                    ErrBadHandle | ErrChannelClosed | ErrPermissionDenied | ErrInternal => {
```

Comment:
@tiziano88 I don't think any of these error statuses (apart from ErrInternal) will be returned any more. Should I still keep them here for completeness?

Reply:
If they should not happen, please capture them explicitly and log an error. If you had such insight to prompt you to write it down in this comment, why disregard this valuable information in the code? It would also make the code self-explanatory.